src/counters/timer.rs

```diff
@@ -46,18 +49,18 @@ impl Timer {
     pub fn resume(&mut self) {
         #[cfg(feature = "profiler")]
         {
-            self.start = Some(instant::now());
+            self.start = Some(web_time::Instant::now());
         }
     }
 
     /// The measured time between the last `.start()` and `.pause()` calls.
     pub fn time(&self) -> f64 {
-        self.time
+        self.time.as_secs_f64() * 1000.0
```

Comment:
Is this ok ?

- https://github.com/rust-lang/rust/issues/122451

Reply:
Let's return the `Duration` here instead.
And add a method `time_ms()` which returns this value.